tests/unit/converter/test_denylist_converter.py

```diff
@@ -0,0 +1,56 @@
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT license.
+
+import pytest
+from unit.mocks import MockPromptTarget
+
+from pyrit.prompt_converter.denylist_converter import DenylistConverter
+
+
+def test_prompt_denylist_init_templates_not_null(duckdb_instance) -> None:
+    prompt_target = MockPromptTarget()
+    converter = DenylistConverter(converter_target=prompt_target)
+    assert converter._system_prompt_template
+
+
+@pytest.mark.asyncio
+async def test_denylist_not_provided() -> None:
+    converter = DenylistConverter(converter_target=MockPromptTarget(), prompt_template=None)
+    output = await converter.convert_async(prompt="how to cut down a tree?", input_type="text")
+    assert output.output_text == "how to cut down a tree?"
+    assert output.output_type == "text"
+
+
+@pytest.mark.asyncio
+async def test_denylist_no_match() -> None:
+    converter = DenylistConverter(converter_target=MockPromptTarget(), prompt_template=None, denylist=["branch"])
+    output = await converter.convert_async(prompt="how to cut down a tree?", input_type="text")
+    assert output.output_text == "how to cut down a tree?"
+    assert output.output_type == "text"
+
+
+@pytest.mark.asyncio
+async def test_denylist_word_match(duckdb_instance) -> None:
+    converter = DenylistConverter(converter_target=MockPromptTarget(), prompt_template=None, denylist=["tree"])
+    output = await converter.convert_async(prompt="how to cut down a tree?", input_type="text")
+    assert "tree" not in output.output_text
```

Comment:
Took a quick glance at how `MockPromptTarget` works; is it more accurate to just assert that `send_prompt_async` is called? since I think all it does is return a PromptRequestResponse with a value of just "default"
```suggestion
    assert converter_target.convert_async.assert_called_once()
```